.changeset/pink-mangos-dream.md

```diff
@@ -0,0 +1,17 @@
+---
+'@twilio-paste/design-tokens': minor
+'@twilio-paste/core': minor
+---
+
+[Design Tokens] Add tokens for new themes `twilio` and `twilio-dark`:
+
+- Aliases: color-palette
+  - Add `palette-red-65`
+  - Add `palette-red-05`
+  - Add `palette-orange-05`
+  - Add `palette-yellow-05`
+  - Add `palette-green-05`
+  - Add `palette-blue-05`
+  - Add `palette-blue-55`
+  - Add `palette-purple-05`
+  - Add `palette-gray-05`
```

Comment:
question: did this alias get added ? I don't think I see it

Reply:
Oh true, Lee added it in a previous PR. Can update that!